src/Api/Serializer/JsonBody.php

```diff
@@ -74,6 +74,10 @@ private function format(Shape $shape, $value)
                     $v = $this->format($values, $v);
                 }
 
+                if (empty($value)) {
```

Comment:
Could this be moved up to the beginning of the case statement and simply return a new `stdclass()`?